addons/csi-aws-ebs/csi-snapshot-controller.yaml

```diff
@@ -48,10 +48,3 @@ spec:
             - "--v=5"
             - "--leader-election=true"
           imagePullPolicy: IfNotPresent
-          resources:
-            limits:
-              cpu: 100m
-              memory: 100Mi
-            requests:
-              cpu: 10m
-              memory: 20Mi
```

Comment:
Some requests and limits would have been nice but I understand that it's probably copied from the host repo.